docs/core-concepts/1205-container-images.md

```diff
@@ -3,16 +3,24 @@ slug: /1205/container-images
 displayed_sidebar: europa
 ---
 
-# Building container images
+# Building docker container images
```

Comment:
Why `docker`? Container is an abstract thing, it can be a docker image or podman image etc no?

Reply:
Okay!